### ISSUE TITLE
How to preview the image in specific HTML tags?

### ISSUE THREAD

**Issue by rikusen0335** [kartoteket/vue-image-upload-resize#11]
I'm sorry I'm just newbie, so this question may you are unconfortable.
So this plugin is working well, but I have a problem that I have no idea how to preview in specific HTML tags.
Is there any way to see the preview image in them?

**Comment by vsirghii** [comment]
Yeah, this helped me too, thanks Svale. What I did to overwrite that width="400" is
<style>
.img-preview {
  max-width: 100% !important;
  height: auto !important;
}
</style>

thus creating preview image fluid and fit nicely into any screen size.

**Comment by svale** [comment]
Thank you for the feedback @vsirghii . I'll take of note that, and we should probably replace the 400px set width with a more responsive max-width.
